#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_gnome-sharp_typo.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix from upsteam SVN r3766 for typo in configure.in.

@DPATCH@
diff -urNad f-spot~/configure.in f-spot/configure.in
--- f-spot~/configure.in	2008-03-15 15:00:06.000000000 +0000
+++ f-spot/configure.in	2008-03-20 17:52:45.000000000 +0000
@@ -74,7 +74,7 @@
 AC_SUBST(F_LIBS)
 
 PKG_CHECK_MODULES(GNOME_SHARP, gnome-sharp-2.0 >= 2.8)
-if pkg-config --atleast-version=2.19.90 gnome-sharp; then
+if pkg-config --atleast-version=2.19.90 gnome-sharp-2.0; then
 	PKG_CHECK_MODULES(GTKHTML_SHARP, gtkhtml-sharp-3.14 >= 2.19.90)
 	GTKHTML_SHARP_PKG="-pkg:gtkhtml-sharp-3.14"
 else
